### ISSUE TITLE
What is this Web3j 5.0.0 version?

### ISSUE THREAD

**Issue by bigman212** [web3j/web3j#1976]
# _Issue_title_
There is a new 5.0.0 Web3J  version in Maven with multiple vulnerabilities 
https://mvnrepository.com/artifact/org.web3j/core/5.0.0

## _Issue_description_
Gradle suggested bumping my 4.10.3 Web3J to a new 5.0.0, but I've checked Maven and 5.0.0 is a very old version.
People can update it by mistake, I just wanted to highlight this issue here.

## _Issue_context_
_Please provide additional information that might be helpful_
<img width="706" alt="Screenshot 2023-10-29 at 17 57 10" src="https://github.com/web3j/web3j/assets/25337696/1eb5b21d-21c4-49fd-ac04-e8408645883d">

**Comment by andras-ballai** [comment]
FYI it isn't actually the newest version, as seen when comparing the dates of releases [here](https://mvnrepository.com/artifact/org.web3j/core). No idea why a previous version was called `5.0.0`, but it's actually 3 years old

**Comment by phraktle** [comment]
I suggest bumping the current library version to 6.x. Based on maven stats, too many projects depend on the 5.0.0 release.

**Comment by phraktle** [comment]
bump, this is still a problem. ~500 projects depend on the 5.0.0 version released in 2020 and only a few on the latest 4.1x series. See https://central.sonatype.com/artifact/org.web3j/crypto/versions

**Comment by AndrewWestberg** [comment]
I agree that this is a major security issue. Any dependency version checkers recommend this old (new) version. I wasted a day of work on a new project before I realized this dependency version was no good.

**Comment by phraktle** [comment]
@gtebrean ?
